README.md

```diff
@@ -0,0 +1,9 @@
+# WP Main
+
+[![Unsupported](https://img.shields.io/badge/Pantheon-Unsupported-yellow?logo=pantheon&color=FFDC28)](https://pantheon.io/docs/oss-support-levels#unsupported)
+
+## About
+Provides functionality and customization for a WordPress site to function in the Pantheon Environment.
+
+## Installation
+Install via Composer: `pantheon-systems/wp-main`.
```

Comment:
This repo is actually a copy of the [mu-plugin](https://wordpress.org/support/article/must-use-plugins/) that currently lives in the [WordPress upstream](https://github.com/pantheon-systems/WordPress/tree/default/wp-content/mu-plugins/pantheon).

There is a [README file](https://github.com/pantheon-systems/wp-main/blob/main/inc/README.md) in the `inc` directory that we could use to replace this one, but add the badge and installation steps to: https://github.com/pantheon-systems/wp-main/blob/main/inc/README.md.

Side note: we actually should pull the history from the origin subdirectory into this repo (via [git filter-repo](https://github.com/newren/git-filter-repo)) and figure out a way to include it in the main upstream, but that's very much outside the scope of CMS-719 😅

Reply:
Good suggestion, fixed!